ext/opentelemetry-ext-dbapi/README.rst

```diff
@@ -20,9 +20,9 @@ Usage
     trace.set_tracer_provider(TracerProvider())
     tracer = trace.get_tracer(__name__)
     # Ex: mysql.connector
-    trace_integration(tracer_provider(), mysql.connector, "connect", "mysql", "sql")
```

Comment:
`tracer_provider` should not be imported anymore, also please check how `get_tracer` is being accessed.